versions/v0.24/modules/en/pages/changelogs/changelogs/v0.9.0.adoc

```diff
@@ -0,0 +1,110 @@
+== "v0.9.0"
+:revdate: 2025-02-10
+:page-revdate: {revdate}
+=== "2024-06-28 10:54:06 +0000 UTC"
+
+// Disclaimer: this file is generated, do not edit it manually.
+
+
+__ (github-actions[bot]) released this 2024-06-28 10:54:06 +0000 UTC - v0.9.0__
+
+
+=== Description
+
+---
+
+++++
+
+<h3><strong><g-emoji class="g-emoji" alias="warning">⚠️</g-emoji> Important:</strong></h3>
+<h4>Starting with Turtles <code>v0.9.0</code>, the process used for importing CAPI clusters into Rancher has changed. We recommend you take a look at the changelog and the instructions provided in the <code>v0.9</code> version of the Turtles book: <a href="https://turtles.docs.rancher.com/" rel="nofollow">https://turtles.docs.rancher.com/</a></h4>
```

Comment:
This should point to linking to the product page instead of the community link. Not sure what the best approach is though since this is HTML and autogenerated (and says don't edit).